File: CS/WindowsApplication1/MyScroolBarPainter.cs
using DevExpress.XtraEditors.ViewInfo;
using DevExpress.Skins;

namespace WindowsApplication1
{
    public class MyScrollBarPainter : SkinScrollBarPainter
    {
        public MyScrollBarPainter(ISkinProvider Skin) : base(Skin) { }

        public override int ThumbMinWidth
        {
            get
            {
                return 8;
            }
        }
    }
}

File: VB/WindowsApplication1/MyScroolBarPainter.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.XtraEditors.ViewInfo
Imports DevExpress.Skins

Namespace WindowsApplication1
	Public Class MyScrollBarPainter
		Inherits SkinScrollBarPainter
		Public Sub New(ByVal Skin As ISkinProvider)
			MyBase.New(Skin)
		End Sub

		Public Overrides ReadOnly Property ThumbMinWidth() As Integer
			Get
				Return 8
			End Get
		End Property
	End Class
End Namespace
